packages/terra-consumer-nav/src/components/modal/Modal.scss

```diff
@@ -30,38 +30,37 @@
   }
 
   .overlay {
-    background-color: var(--terra-consumer--overlay-background, #64696c);
-    opacity: 1;
+    background-color: var(--terra-consumer--overlay-background, #64696c) !important;
+    opacity: 1 !important;
     @media screen and (max-width: $terra-medium-breakpoint) {
       height: 100vh;
       width: 100vw;
     }
   }
 
-  .close-button {
-    background: none;
-    border: 0;
-    color: var(--terra-consumer--modal-close-button-color, #fff);
-    font-size: 16px;
+  .close-button-container {
     margin: 0;
     padding: 0;
     position: absolute;
     right: 20px;
-    top: 5px;
+    top: 0;
```

Comment:
what's difference between not adding `top` and  adding `top: 0` ?

Reply:
removed it